packages/chat/src/ChatClient.ts

```diff
@@ -537,6 +538,19 @@ export class ChatClient extends IrcClient {
 		[channel: string, user: string, forwardInfo: ChatCommunityPayForwardInfo, msg: UserNotice]
 	> = this.registerEvent();
 
+	/**
+	 * Fires when a user sends an announcement (/announce) to a channel.
+	 *
+	 * @eventListener
+	 * @param channel The channel the action was sent to.
```

Comment:
No actions here, only announcements

Reply:
Changed it. I thought it was fitting of the docs